server/src/main/scala/com/kristofszilagyi/pipelineboard/controllers/Config.scala

```diff
@@ -17,6 +19,8 @@ object JobConfig {
   implicit val urlUserFormat: YamlFormat[UserRoot] = wrappedYamlString(s => UserRoot(RawUrl(Uri.parse(s))))(_.u.rawString)
   implicit val urlRestFormat: YamlFormat[RestRoot] = wrappedYamlString(s => RestRoot(RawUrl(Uri.parse(s))))(_.u.rawString)
   implicit val groupNameFormat: YamlFormat[GroupName] = wrappedYamlString(GroupName.apply)(_.s)
+
+  implicit val finiteDurationFormat: YamlFormat[FiniteDuration] = wrappedYamlNumber(n => FiniteDuration.apply(n.longValue(), NANOSECONDS))(_.toNanos)
```

Comment:
so what changed here is that it's ns now and not minutes, right? While this is more natural from the point of view of FiniteDuration I don't think I like it. I would have preferred if the config was called `fetchFrequencyMinutes` and it was a number or you can define a `Minutes` case class. Or you could have it called `fetchFrequency` but then it should mandate units in the value, so you could write `fetchFrequency: 60 seconds`.` fetchFrequency: 600000000000` looks quite ugly and not very readable as the user doesn't know the unit.

Reply:
Good thinking - done (Minutes case class)